app/src/main/java/org/gdg/frisbee/android/arrow/NotificationHandler.java

```diff
@@ -7,53 +7,51 @@
 import android.content.Intent;
 import android.support.v4.app.NotificationCompat;
 
-import org.gdg.frisbee.android.R;
-import org.gdg.frisbee.android.chapter.MainActivity;
+import org.gdg.frisbee.android.Const;
+import org.gdg.frisbee.android.eventseries.TaggedEventSeries;
+import org.gdg.frisbee.android.eventseries.TaggedEventSeriesActivity;
 import org.gdg.frisbee.android.utils.PrefUtils;
-import org.joda.time.DateTime;
-import org.joda.time.DateTimeZone;
 
 public class NotificationHandler {
 
     private final Context context;
-    public static final DateTime SUMMIT_DATE_TIME = new DateTime(2016, 5, 17, 15, 0, DateTimeZone.UTC);
+    private final TaggedEventSeries eventSeries;
 
-    public NotificationHandler(Context context) {
+    public NotificationHandler(Context context, TaggedEventSeries eventSeries) {
         this.context = context;
+        this.eventSeries = eventSeries;
     }
 
     public boolean shouldSetAlarm() {
-        return !PrefUtils.isSummitNotificationSent(context)
-            && SUMMIT_DATE_TIME.isAfterNow();
+        return !PrefUtils.isTaggedEventSeriesNotificationSet(context, eventSeries)
+            && eventSeries.getStartDateInMillis().isAfterNow();
     }
 
     public void setAlarmForNotification() {
         AlarmManager am = (AlarmManager) context.getSystemService(Context.ALARM_SERVICE);
 
         Intent intent = new Intent(SummitNotificationReceiver.ACTION_SUMMIT_NOTIFICATION);
+        intent.putExtra(Const.EXTRA_TAGGED_EVENT, eventSeries);
 
-        // Tuesday 8 AM PST in the morning
-        am.set(AlarmManager.RTC_WAKEUP, SUMMIT_DATE_TIME.getMillis(),
```

Comment:
We usually set the start date of an event series as the UTC midnight, right? Should we make this little bit more intelligent to set the morning in user's timezone?
Does it worth it?